src/Styling/LandingPage.css

```diff
@@ -10,8 +10,22 @@
         0 100px 80px rgba(0, 0, 0, 0.12);
 }
 
+.landing-page p {
+    color: #F56F60;
+}
+
+.landing-welcome {
+    position: relative;
+    margin: 50px auto;
+}
+
+.landing-welcome img {
```

Comment:
Can these be tailwind classnames?

Reply:
I made the changes. Let me know if there's anything else you want me to change.

Reply:
Theres still merge confilcts on this branch, could you fix those?

Reply:
Ahh ok I'm sorry. Fixed the merge conflicts!